packages/react-components/lib/robots/robot-table.tsx

```diff
@@ -160,7 +160,7 @@ export function RobotTable({
               <TableCell>Robot Name</TableCell>
               <TableCell>Start Location</TableCell>
               <TableCell>Destination</TableCell>
-              <TableCell>End Time</TableCell>
+              <TableCell>Duration</TableCell>
```

Comment:
Is the duration in seconds? If so we should make the units clear. Maybe `Duration (s)`? Or we could leave the column header as suggested and append the `s` or `seconds` after the integer data.

Reply:
Added an `s` after the data to be consistent with the existing format of other columns